chia/server/server.py

```diff
@@ -131,6 +131,7 @@ class ChiaServer:
     ssl_client_context: ssl.SSLContext
     node_id: bytes32
     exempt_peer_networks: list[Union[IPv4Network, IPv6Network]]
+    class_for_type: dict[NodeType, Any]
```

Comment:
Shouldn't this be annotated as `dict[NodeType, ApiProtocol]`?

Reply:
Might be worth having a type alias for this?  Would help lead readers back to the one origin of it all more quickly.